src/templates/profile.ts

```diff
@@ -2,16 +2,67 @@
  * Returns a usecase header with filled in `name` and `version`.
  */
 export function header(name: string, version: string): string {
-  return `name = "${name}"
+  return `"""
+${name} Profile
+Profile description
```

Comment:
_(also applies to `usecase` block)_

With respect to `prepare:map` where we offer numbered list of _“to dos”_, what if we just say these descriptions are `TODO`? It's a known convention for “stuff you need to resolve later”. Even if it becomes published, it's less awkward as the consumer would also recognize a TODO.

Another level would be using the actual profile name:

```suggestion
TODO: What “Address Ip Geolocation” does
```